client/state/plugins/premium/selectors.js

```diff
@@ -92,3 +98,21 @@ export const getNextPlugin = function( state, siteId, whitelist = false ) {
 	}
 	return plugin;
 };
+
+export const getPluginKeys = createSelector(
+	( state, siteId, whitelist = false ) => {
+		const pluginList = getPluginsForSite( state, siteId, whitelist );
+
+		return pluginList.reduce( ( keys, plugin ) => {
+			const key = get( plugin, 'key', null );
+			const slug = get( plugin, 'slug', null );
```

Comment:
No need for the `null` fallbacks I think -- they'll be `undefined` and thus falsey (which is all that matters for the check below).

Reply:
Cleaned up in 04e53cd2eceafbb079906efc11929b4889d37e72 👍